i18n/changed.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""
+Determine if the source translation files are up-to-date.
+"""
+from __future__ import print_function
+from subprocess import CalledProcessError
+import sys
+
+from i18n import Runner
+from i18n.execute import execute
+
+
+class Changed(Runner):
+    """
+    Class used to check if the source translation files are up-to-date
+    """
+    def run(self, args):
+        """
+        Main entry point of script
+        """
+        changes_detected = self.detect_changes()
+        message = self.get_message(changes_detected)
+        print(message)
+        sys.exit(1)
```

Comment:
1) I replaced this line with
    if changes_detected:
        sys.exit(1)
    sys.exit(0)
Is this the proper way?
2) Why is 1) not enough? I'm not sure what to do here since I don't know why it needs to be done. My best guess is that the SystemExit exception must be caught something like this
try:
    self.run(args)
except SystemExit as exit_code:
    sys.exit(exit_code)